src/ObjCBindings/ExportTag.cs

```diff
@@ -134,6 +145,11 @@ public enum Property : Int64 {
 		/// </summary>
 		Transient = 1 << 8,
 
+		/// <summary>
+		/// If this flag is applied to a property, we do not generate a NSString for
+		/// marshalling the property.
+		/// </summary>
+		PlainString = 1 << 9,
```

Comment:
it would make sense to use the same bit value for the same flag at some point

Reply:
yep, atm is more of a work in progress so flags are a mismatch. I should coordinate them at some point. Issue is that bgen used to have 35 attrs, so I am cleaning along the way..